app/mailers/user_mailer.rb

```diff
@@ -51,7 +51,7 @@ def invitation(invitation_id)
     @user_name = (@invitation.creator.is_a?(User) ? @invitation.creator.login : '')
     mail(
       to: @invitation.invitee_email,
-      subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Invitation"
+      subject: t(".invitation.subject", app_name: ArchiveConfig.APP_SHORT_NAME)
```

Comment:
Because it's not in the view file, I think this needs to be `.user_mailer.invitation.subject` or else there will be a missing translation error.